plugin/emacs/flycheck-quicklintjs.el

```diff
@@ -0,0 +1,101 @@
+;;; flycheck-quicklintjs --- Quick Lint JS flycheck support   -*- lexical-binding: t; -*-
+
+;;; Commentary:
+
+;; This flycheck exetension configures flycheck to run Quick Lint JS
+;; to run as a checker.
+;;
+;; Example usage in your init.el:
+;;
+;; (require 'flycheck-quicklintjs)
+;;
+;; ;; Optionally you can setup flycheck with the expected
+;; ;; quick lint js experience:
+;; (defun my-flycheck-quicklintjs-setup ()
+;;   "Configure flycheck-quicklintjs for better experience."
+;;
+;;   ;; Enable flycheck
+;;   (unless (bound-and-true-p flycheck-mode)
+;;     (flycheck-mode))
+;;
+;;   ;; Use quick-lint-js by default when in 'js-mode`
+;;   (flycheck-select-checker 'javascript-quicklintjs)
+;;
+;;   ;; Remove any delay after a change in buffer to run checkers.
+;;   ;; The default is 0.5 (500ms)
+;;   (setq-local flycheck-idle-change-delay 0)
+;;
+;;   ;; Arguments to quick-lint-js program
+;;   ;; (setq-local flycheck-quicklintjs-args "--output-format=gnu-like")
```

Comment:
Blocking: Don't suggest that the user change the `--output-format`. That'll break our plugin's parsing!

quick-lint-js doesn't yet have any options which makes sense for a user to customize. Therefore, let's not suggest any options.